packages/devtools_app/lib/src/shared/connected_app.dart

```diff
@@ -248,13 +250,16 @@ class AppState {
     cache._clear();
   }
 
+  // TODO(polina-c): add explanation for variables.
   ValueListenable<List<DartObjectNode>> get variables => _variables;
   final _variables = ValueNotifier<List<DartObjectNode>>([]);
   void setVariables(List<DartObjectNode> value) => _variables.value = value;
 
   ValueListenable<bool> get isPaused => _isPaused;
   final _isPaused = ValueNotifier<bool>(false);
-  void setPaused(bool value) => _isPaused.value = value;
+
+  /// This setter should be invoked only by debugger.
+  void setPausedOnBreakpoint(bool value) => _isPaused.value = value;
```

Comment:
Fyi @kenzieschmoll what pattern should devtools standardize on for this case? Generally it is violating the Dart style guide to have members that start with the word set but I do see the benefit of this approach relative to having the getter be type ValueNotifier.

Reply:
I cannot find the PR, but several months ago we had this same discussion. The Dart style guide only says to avoid starting methods with "get" but does not say to do so for "set". I remember this because I thought the same thing as you, to avoid "get" and "set", but couldn't find the style guide evidence to support that. Can you find a reference to where we are getting the idea to avoid "set" as well?

Reply:
maybe from other languages: https://dev.to/scottshipp/avoid-getters-and-setters-whenever-possible-c8m